content/en/cloud_cost_management/anomalies.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Anomalies Page
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/monitors"
+  tag: "Monitors"
+  text: "Create Cost Monitors"
+---
+
+## Overview
+
+Datadog Cloud Cost Management (CCM) continuously monitors your environment to identify and prioritize unexpected cost changes, so that you can detect, share, investigate, and resolve cost anomalies. Cost anomalies are currently available for AWS, and does not require any additional set up after CCM is set up.
+
+## View cost anomalies
+
+You can filter to Active, Past, or Resolved anomalies:
+- **Active**: An anomaly is **Active** if there were anomalous costs in the last day (the last full day of cost data, which is often 2-3 days prior). 
+- **Past**: If an anomaly lasts more than 7 days, or the algorithm detects that costs are no longer anomalous, the anomaly is moved to the **Past** tab. Past anomalies can be useful to report on, but are often less urgent and actionable.
+- **Resolved**: If you Resolve an anomaly, the anomaly can be found in the **Resolved** tab.
+
+Each anomaly explains how much more costs were than expected, the specific cloud service name (ex: rds), usage type (ex: db.r6g.12xl), and cloud accounts where the anomaly occured. The anomaly also shows what expected costs would have been in the time frame, with a graph that shows the cost pattern over the past 1 month.
+
+Anomalies with the most unexpected costs are at the top, so that it is easier to take action on anomalies with the most impact first.
+
+{{< img src="cloud_cost/anomalies/cost anomalies - list.png" alt="List of cost anomalies automatically detected" style="width:100%;" >}}
+
+## Take action on anomalies
+
+Click an anomaly to view the services, teams, environments, and resource IDs that may be driving the cost anomaly.
+
+Investigate the anomaly further, and by any additional dimensions, by viewing the costs in Explorer or saving the query to a Notebook. You can also send the anomaly, Explorer link, or Notebook to the associated service owners or teams, so they can help provide context for why the anomaly occured, and if it's expected or not.
+
+You can also create a cost anomaly monitor to get alerted of similar cost anomalies in the future.
+
+{{< img src="cloud_cost/anomalies/cost anomalies - side panel" alt="In the side panel you can take action on your cost anomaly" style="width:100%;" >}}
```

Comment:
Just flagging that this image seems to be broken

Reply:
ty for flagging!